mitosheet/src/components/taskpanes/Import/CSVImport.tsx

```diff
@@ -184,10 +187,11 @@ function CSVImport(props: CSVImportProps): JSX.Element {
     // and not for importing the file
     const [fileMetadata, setFileMetadata] = useState<CSVFileMetadata | undefined>(undefined);
     const {params, setParams, loading, edit, editApplied, error} = useSendEditOnClick(
-        () => getDefaultParams(props.fileName),
+        () => getDefaultCSVParams(props.fileName),
         StepType.SimpleImport,
         props.mitoAPI, props.analysisData,
-        {allowSameParamsToReapplyTwice: true}
+        {allowSameParamsToReapplyTwice: true},
+        props.updateImportEdit
```

Comment:
Probably the same architectural points here. I know it's a more code, but in practice it feels like we'll end up with a much better breakdown that doesn't really complicate logic as much.

Reply:
I think I'd want to try it at least and see how it goes. It might not be the right move, but I I'd want to see.